cloudbuild.yaml

```diff
@@ -31,10 +31,15 @@ steps:
         export CONFIG=${_CONFIG}
         export PROJECT_ID=${PROJECT_ID}
         export IMAGE=${_IMAGE}
-        './cloudbuild/build.sh'
+        export CUSTOM_INSTANCE_NAME=${REPO_NAME}-${BRANCH_NAME}-at-${SHORT_SHA}
```

Comment:
`REPO_NAME`, `BRANCH_NAME` and `SHORT_SHA` variables are only available when builds are run from a commit trigger. If I just run `gcloud builds submit --project=appsembler-devstack-30` this results in an instance name of `---at-` and that causes the build to fail. Being able to trigger builds without pushing commits is a much nicer workflow for iterating on the build.